core/src/main/scala/kafka/log/LocalLog.scala

```diff
@@ -370,11 +370,11 @@ class LocalLog(@volatile private var _dir: File,
         throw new OffsetOutOfRangeException(s"Received request for offset $startOffset for partition $topicPartition, " +
           s"but we only have log segments upto $endOffset.")
 
-      if (startOffset == maxOffsetMetadata.messageOffset)
+      if (startOffset == maxOffsetMetadata.messageOffset) {
```

Comment:
We don't use {} for single line statement elsewhere in this file. So, we want to make this consistent here.